mlir/utils/jenkins/Jenkinsfile

```diff
@@ -208,7 +214,7 @@ pipeline {
                     }
                     post {
                         always {
-                            cleanWs()
+                            deleteDir()
```

Comment:
Come to think of it, wouldn't it make sense to not delete the whole directory but just the build?

Reply:
Why? We've finished all tasks. It's OK to delete the whole directory and leave.

Reply:
On the other hand, that means we'd need to re-clone everything, no?

Reply:
it's better to clear the workspace folder most of the times after certain number of  builds the workspace folder gets corrupted somehow causing the build to fail, Besides for mlir the job does shallow clone so the cloning is faster

Reply:
When we validated that `deleteDir()` can work, can we fall back to `cleanWs()` @okakarpa? I think it is still good for this task to go async as long as it still does its job.

Reply:
Most likely cleanWs() should work as well since we are using the same uid as jenkins for deleting workspace 
once the PR is updated i can review the node workspace again